src/server/es/index.js

```diff
@@ -226,11 +226,16 @@ class ES {
           }
           const fields = doc._source.array;
           fields.forEach((field) => {
-            if (!this.fieldTypes[index][field]) {
-              const errMsg = `[ES.initialize] wrong array entry from config: field "${field}" not found in index ${index}, skipped.`;
-              log.error(errMsg);
-              return;
-            }
+            const nodes = field.split('.');
+            let nestedObject = this.fieldTypes[index];
+            nodes.forEach((n) => {
```

Comment:
I think you should just rewrite this loop through `nodes` using a plain for loop so you can break out if a `node` doesn't meet the creatia. You can't really break out or return easily from a `forEach` callback, so the `field` will stull be added